draft-ietf-tls-external-psk-importer.md

```diff
@@ -41,22 +59,20 @@ TLS 1.3.
 
 # Introduction
 
-TLS 1.3 {{!RFC8446}} supports pre-shared key (PSK) authentication, wherein PSKs
-can be established via session tickets from prior connections or externally via some out-of-band
-mechanism. The protocol mandates that each PSK only be used with a single hash function.
-This was done to simplify protocol analysis. TLS 1.2 {{!RFC5246}}, in contrast, has no such requirement, as
-a PSK may be used with any hash algorithm and the TLS 1.2 PRF. This means that external PSKs
-could possibly be re-used in two different contexts with the same hash functions during key
-derivation. Moreover, it requires external PSKs to be provisioned for specific hash
-functions.
-
-To mitigate these problems, external PSKs can be bound to a specific hash function when used
-in TLS 1.3, even if they are associated with a different KDF (and hash function) when provisioned. This document
-specifies an interface by which external PSKs may be imported for use in a TLS 1.3 connection
-to achieve this goal. In particular, it describes how KDF-bound PSKs can be differentiated by
-different hash algorithms to produce a set of candidate PSKs, each of which are bound to a specific
-hash function. This expands what would normally have been a single PSK identity into a set of
-PSK identities. However, it requires no change to the TLS 1.3 key schedule.
+TLS 1.3 supports PSK-based handshakes, wherein PSKs can be established either via session tickets from prior TLS connections or via an out-of-band mechanism.
+TLS 1.3 mandates that each PSK be associated with a single hash function.
+This makes it easier for formal analysis to exclude cross-protocol attacks where the same key is used in multiple different contexts.
+TLS 1.2 also supports PSK-based handshakes, but does not require they be provisioned with a single hash function.
```

Comment:
For the above "TLS 1.3 mandates that each PSK be associated", should "be" -> "is"?
For this "does not require they be provisioned", "they" -> "them to" or "these to"?